### DISCUSSION TITLE
关于PP-OCRv4训练速度问题

### DISCUSSION THREAD

**Discussion by jadehh** [PaddlePaddle/PaddleOCR#12231] [Q&A]
在GPU环境中,训练en_PP-OCRv4_rec.yml,速度慢
系统环境/System Environment：windows GPU
版本号/Version：PaddlePaddle：2.3.2 
显卡:NVIDIA 2080TI,CUDA版本:8.6
![0cc8475b55a0710204f2b42e73ef2fb](https://github.com/PaddlePaddle/PaddleOCR/assets/20532462/85388cea-dd16-416d-accc-a656317be02c)
这样的速度正常吗,GPU使用率基本维持在100%

**Comment by UserWangZz** [comment]
数据量多少？batchsize设置多少？

**Comment by nissansz** [comment]
哪个配置的SVTR配置训练速度最快，准确率最高？
